web/src/styles/pixel-skill-card.css

```
@@ -0,0 +1,621 @@
+}
+.pixel-skill-card__stats dd {
+  margin: 0;
+}
+.pixel-skill-card__visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.10.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)